[2022-04-14 07:31:38,278] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test_1_task_10s.task manual__2022-04-14T07:31:37.582832+00:00 [queued]>
[2022-04-14 07:31:38,304] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: test_1_task_10s.task manual__2022-04-14T07:31:37.582832+00:00 [queued]>
[2022-04-14 07:31:38,304] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 07:31:38,305] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-14 07:31:38,305] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-14 07:31:38,332] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): task> on 2022-04-14 07:31:37.582832+00:00
[2022-04-14 07:31:38,339] {standard_task_runner.py:52} INFO - Started process 4740 to run task
[2022-04-14 07:31:38,341] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test_1_task_10s', 'task', 'manual__2022-04-14T07:31:37.582832+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_one_task_10s.py', '--cfg-path', '/tmp/tmpzxdf2eob', '--error-file', '/tmp/tmp9ym9dcup']
[2022-04-14 07:31:38,342] {standard_task_runner.py:80} INFO - Job 19: Subtask task
[2022-04-14 07:31:38,415] {logging_mixin.py:109} INFO - Running <TaskInstance: test_1_task_10s.task manual__2022-04-14T07:31:37.582832+00:00 [running]> on host 877f390e4836
[2022-04-14 07:31:38,494] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=test_1_task_10s
AIRFLOW_CTX_TASK_ID=task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-14T07:31:37.582832+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-14T07:31:37.582832+00:00
[2022-04-14 07:31:48,504] {python.py:175} INFO - Done. Returned value was: None
[2022-04-14 07:31:48,516] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=test_1_task_10s, task_id=task, execution_date=20220414T073137, start_date=20220414T073138, end_date=20220414T073148
[2022-04-14 07:31:48,575] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-14 07:31:48,610] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
